refactron/cicd/pr_integration.py

```
@@ -0,0 +1,212 @@
+        Returns:
+            Markdown formatted comment
+        """
+        lines = []
+        lines.append(f"### {self.level.upper()}: {self.rule_id or 'Issue'}")
+        lines.append("")
+        lines.append(self.message)
+        lines.append("")
+
+        if self.suggestion:
+            lines.append("**Suggestion:**")
+            lines.append(f"```python")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)